install/__init__.py

```diff
@@ -30,13 +34,23 @@ def install(pkg, use_pep517=None, requirements=None, pip_options=None, install_o
         pkg: Name of the package or requirements.txt file as a string, you can also use version specifiers like requests==1.2.3
         use_pep517: Optional boolean to force --use-pep517/--no-use-pep517
         requirements: Optional boolean if a requirements.txt was specified
-        pip_options: Optional arbitary list of global options to pass to pip 
+        pip_options: Optional arbitary list of global options to pass to pip
         install_options: Optional arbitary list of install options to pass to pip install
     """
+    # exit fast if pkg already installed
+    if sys.version[0] == '3':
```

Comment:
Do we need to gate this behind py3? py2.7 has import_module.

Reply:
In that case, we can also include python2. I have never really used Python2. About the ..., it's the same as pass yeah, just one keystroke fewer:), no particular reason.